netspeed/data/org.mate.panel.applet.netspeed.gschema.xml.in

```diff
@@ -16,9 +16,9 @@
       <description>If true, show speed in bits instead of bytes.</description>
     </key>
     <key name="show-all-addresses" type="b">
-      <default>false</default>
+      <default>true</default>
```

Comment:
I don't think it should be enabled by default

Reply:
Why not?

Reply:
Most users use IPv4 and set ignore in the IPv6 addressing mode, which configures the interface to use link-local address fe80::/10 + mac.

Reply:
Are you sure? Most users leave it on automatic, I believe. And many ISPs offer IPv6 nowadays. And IPv6 is becoming more and more important. Thus, it should show both by default. Whoever sets IPv6 to "ignore" should manually disable the checkbox if one doesn't like to see the link-local address. And whoever sets IPv6 to "disabled" will not even see the link-local address.

Reply:
32% users use IPv6 according to google statistics:
https://www.google.com/intl/en/ipv6/statistics.html

Reply:
Yes, 32% is already many ISPs. T-Mobile USA is even IPv6-only. If you tether with your iPhone, you won't get an IPv4 address at all.